src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -53,7 +53,7 @@ public void execute_personAcceptedByModel_addSuccessful() throws Exception {
     }
 
     /**
-     * tests for duplicate persons
+     * Tests for duplicate persons.
```

Comment:
Slightly confused why the need to update the prevous code base person model. Is it failing checkstyle?